Yoshi-s-Yodelers/BookIt
# Extension init

Added a basic hello-world extension popup page

---
to test:
1. Pull/Install/Build webpack
2. Go to chrome://extensions
3. Turn on developer mode
4. Click 'load unpacked' and point it at the public directory

- [x] Success condition: When you click the extension in your chrome browser, it should open a small popup with the words 'Hello World'
